assets/css/translation.css

```diff
@@ -30,27 +70,41 @@ html[dir='rtl'] .languageVariant {
     cursor: pointer;
 }
 
-html[dir='ltr'] #srcLanguages {
-    right: auto;
+html[dir='ltr'] .languageVariant {
+    font-size: 12px;
+    padding-left: 2.5em;
 }
 
-html[dir='ltr'] #dstLanguages {
-    left: auto;
-    right: 0;
+html[dir='rtl'] .languageVariant {
+    font-size: 12px;
```

Comment:
I think this `font-size: 12` is common and can be factored out?

Reply:
Yup. :)